src/components/GraphFilter/GraphFilter.tsx

```diff
@@ -104,12 +96,11 @@ export default class GraphFilter extends React.PureComponent<GraphFilterPropsRea
             options={GraphFilter.GRAPH_TYPES}
           />
           <Toolbar.RightContent>
-            <GraphRefreshWithDefaultOptions
+            <GraphRefreshContainerDefaultRefreshIntervals
               id="graph_refresh_container"
               disabled={this.props.disabled}
               handleRefresh={this.handleRefresh}
-              graphDuration={this.props.graphDuration}
-              onUpdateGraphDuration={this.handleDuration}
+              duration={store.getState().userSettings.duration}
```

Comment:
I think, you don't need this property, since `GraphRefreshContainerDefaultRefreshIntervals` is overriding this internally (you already changed that component to pass the duration). So, no need to import the store.

Reply:
Excellent catch, I  created KIALI-1894 for this, and will have a PR soon.